ui/src/app/shared/services/parser.ts

```diff
@@ -557,15 +529,16 @@ class InternalParser {
                     }
                 } else {
                     const streamdef = this.eatStream(lineNumber);
-                    console.log('JSParse: parsed to stream definition: ' + JSON.stringify(streamdef));
                     // streamDef = {"apps":[{"name":"time","start":0,"end":4},{"name":"log","start":7,"end":10}]}
```

Comment:
Can we remove these commented lines.  If so, I can do it on merge.

Reply:
ok